src/ripple/rpc/tests/Subscribe.test.cpp

```diff
@@ -264,6 +282,31 @@ class Subscribe_test : public beast::unit_test::suite
             expect(jv[jss::status] == "success");
         }
 
+        {
+            // Accept a ledger
+            env.close();
+
+            // Check stream update
+            expect(wsc->findMsg(5s,
+                [&](auto const& jv)
+                {
+                    return jv[jss::type] == "validationReceived" &&
+                        jv[jss::validation_public_key] ==
+                            "n9MvFGjgv1kYkm7bLbb2QUwSqgzrQkYMYHXtrzN8W28Jfp2mVihq" &&
+                        jv[jss::ledger_hash] ==
+                            to_string(env.closed()->info().hash) &&
+                        jv[jss::ledger_index] ==
+                            to_string(env.closed()->info().seq) &&
+                        jv[jss::signature] ==
+                            "30440220461C608DF851A067A268ACAE3A42EDDDA40A3C7F1F40FBDCE9E84D4DD2333AC902203C33F332CC49B17E72DF74D2E9F3863F06AF4A82170058385B36B35CA345603A" &&
```

Comment:
Should I just check that the `signature` exists, since the value seems likely to change?